ros_ws/src/hardware/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(hardware)
+find_package(catkin REQUIRED)
+catkin_metapackage()
```

Comment:
You said that this package cannot be a metapackage because it contains a parameter file. Should this line then be here?